transformer_engine/__init__.py

```diff
@@ -20,3 +18,6 @@
     from . import tensorflow
 except ImportError as e:
     pass
+
+from . import common
```

Comment:
`transformer_engine.common` does initialization, like loading the C extensions, so we should import it before the framework extensions.